infra/api/service/copy_oracle_data.tf

```diff
@@ -0,0 +1,89 @@
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sfn_state_machine
+resource "aws_sfn_state_machine" "copy_oracle_data" {
+
+  name     = "${local.service_name}-copy-oracle-data"
+  role_arn = module.service.task_role_arn
+
+  definition = jsonencode({
+    "StartAt" : "ExecuteECSTask",
+    "States" : {
+      "ExecuteECSTask" : {
+        "Type" : "Task",
+        # docs: https://docs.aws.amazon.com/step-functions/latest/dg/connect-ecs.html
+        "Resource" : "arn:aws:states:::ecs:runTask.sync",
+        "Parameters" : {
+          "Cluster" : module.service.cluster_arn,
+          "TaskDefinition" : module.service.task_definition_arn,
+          "LaunchType" : "FARGATE",
+          "NetworkConfiguration" : {
+            "AwsvpcConfiguration" : {
+              "Subnets" : data.aws_subnets.private.ids,
+              "SecurityGroups" : [module.service.app_security_group_id],
+            }
+          },
+          "Overrides" : {
+            "ContainerOverrides" : [
+              {
+                "Name" : local.service_name,
+                "Environment" : [
+                  {
+                    "Name" : "FLASK_APP",
+                    "Value" : "src.app:create_app()",
+                  }
+                ]
+                "Command" : [
+                  "poetry",
+                  "run",
+                  "flask",
+                  "data-migration",
+                  "copy-oracle-data",
```

Comment:
The command added in https://github.com/HHS/simpler-grants-gov/pull/1243

Reply:
Do we want to (can we?) use the make command instead?

Reply:
We can. I just very slightly prefer it this way.